app.py

```diff
@@ -46,56 +45,71 @@ def main():
         
     if choice == "Home":
         # Let's set the title of our awesome web app
-        st.title('Auto Vaidya')
-        # Now setting up a header text
-        st.subheader("Automating Healthcare one problem at a time")
+        img  = Image.open('assets/logo.png')
+        covid_img = Image.open('assets/coronavirus.jpg')
+        covid_img = covid_img.resize((100,100))
+        diabetic_img = Image.open('assets/diabetic_retinopathy.jpeg')
+        diabetic_img = diabetic_img.resize((100,100))
+        heart_img = Image.open('assets/Heart-graphic.jpg')
+        heart_img = heart_img.resize((100,100))
+        pneu_img = Image.open('assets/pneumonia.jpg')
+        pneu_img = pneu_img.resize((100,100))
+        skin_img = Image.open('assets/skin-cancer.jpg')
+        skin_img = skin_img.resize((100,100))
+        st.image(img)
+        st.image([covid_img,diabetic_img,heart_img,pneu_img,skin_img],caption=["COVID-19 Detection","Diabetic Retinopathy","Heart Disease Prediction","Pneumonia Detection","Skin Cancer Detection"])
+        st.write("Illustration of the Web Application:")
+        st.video('assets/demo.mp4')
```

Comment:
Can you please put a small recording of the app with this change in the description of this Pull Request

Reply:
Yes, [Here](https://drive.google.com/file/d/1EkHoW10z5WJbptyyZpnVMhd73R4HzLve/view?usp=sharing) is the link.

Reply:
Thanks